lib/hooks/fetchGithubIssueInfo.ts

```diff
@@ -0,0 +1,35 @@
+import { Octokit } from "octokit";
+import { supabase } from "lib/utils/supabase";
+
+interface GhIssueInfoResponse {
+  data: GhIssueInfoResponse;
+  status: number;
+  headers: {};
+}
+const fetchGithubIssueInfo = async (orgName: string | null, repoName: string | null, issueId: string | null) => {
+  const sessionResponse = await supabase.auth.getSession();
+  const sessionToken = sessionResponse?.data.session?.provider_token;
+  try {
+    const octokit = new Octokit({ auth: sessionToken });
+
+    const response: GhIssueInfoResponse = await octokit.request("GET /repos/{orgName}/{repoName}/issues/{issueId}", {
+      orgName,
+      repoName,
+      issueId,
+    });
```

Comment:
Wouldn't it be better to use octokit's issues API instead of using the raw rest APIs?

https://octokit.github.io/rest.js/v20#issues-get

```js
octokit.rest.issues.get({
  owner,
  repo,
  issue_number,
});

Reply:
I just went followed what was already done for PRs, but maybe we can refactor as part of another PR.